torch/onnx/symbolic_caffe2.py

```diff
@@ -0,0 +1,57 @@
+import importlib
+from inspect import getmembers, isfunction
+from torch.onnx.symbolic_helper import parse_args
+import torch.onnx.symbolic_registry as sym_registry
+import torch.onnx.symbolic_helper as sym_help
+
+def register_quantized_ops(domain, version):
+    # Register all the non-quantized ops
+    sym_registry.register_version('', version)
+    # Register all quantized ops
+    module = importlib.import_module('torch.onnx.symbolic_caffe2')
+    sym_registry._symbolic_versions['caffe2'] = module
+    quant_version_ops = getmembers(sym_registry._symbolic_versions['caffe2'])
+    for op in quant_version_ops:
+        if isfunction(op[1]) and not sym_registry.is_registered_op(op[0], domain, version):
+            sym_registry.register_op(op[0], op[1], domain, version)
+
+def linear_prepack(g, input, weight):
```

Comment:
actually, for some ops, like relu, we need check whether inputs are quantized or not, and dispatch in the symbolics here. We should override the old symbolics since we already include the regular paths.